framework/log/Logger.php

```diff
@@ -41,27 +46,33 @@
  * @author Qiang Xue <qiang.xue@gmail.com>
  * @since 2.0
  */
-class Logger extends Component
+class Logger extends Component implements LoggerInterface
 {
+    use LoggerTrait;
+
     /**
      * Error message level. An error message is one that indicates the abnormal termination of the
      * application and may require developer's handling.
+     * @deprecated since 2.1.0 Use [[LogLevel::ERROR]] instead.
      */
-    const LEVEL_ERROR = 0x01;
+    const LEVEL_ERROR = LogLevel::ERROR;
```

Comment:
It's 2.1 so I think we should drop these local constants and use `LogLevel` directly.